packages/core/src/kernelTemplate.js

```diff
@@ -22,21 +33,10 @@
       scuttleGlobalThisExceptions,
     } = __lavamoatSecurityOptions__
 
-    // identify the globalRef
-    const globalRef = (typeof globalThis !== 'undefined') ? globalThis : (typeof self !== 'undefined') ? self : (typeof global !== 'undefined') ? global : undefined
-    if (!globalRef) {
-      throw new Error('Lavamoat - unable to identify globalRef')
-    }
-
-    // polyfill globalThis
-    if (globalRef && !globalRef.globalThis) {
-      globalRef.globalThis = globalRef
-    }
-
```

Comment:
I'm assuming this https://github.com/LavaMoat/LavaMoat/pull/459/files goes here as a result of merging/rebasing?

Reply:
Indeed. The lavapack parts were not included there and are covered in #461.